libraries/chain/controller.cpp

```diff
@@ -1980,6 +1980,7 @@ struct controller_impl {
    void set_finalizers_impl(const finalizer_set& fin_set) {
       // TODO store in chainbase
       current_finalizer_set = fin_set;
+      ++current_finalizer_set.generation;
```

Comment:
This method sets finalizer set and increments the generation. Should its name be changed to reflect the two tasks?

Reply:
This will be going away in the next PR.